Sources/Orbit/Support/TextConcatenation/TextRepresentable.swift

```diff
@@ -0,0 +1,11 @@
+import SwiftUI
+
+public protocol SwiftUITextRepresentable {
+    func asText(configuration: ContentSizeCategory) -> SwiftUI.Text?
```

Comment:
Hmm I don't recall many "asSomething"s in Swift... I was originally going to write that this should be expressed as an initializer, but that would be an initializer on `SwfitUI.Text` 🤔 

Something just doesn't feel "Swifty" about this, but I can't think of a better way to do it at the moment 😄

Reply:
renamed to `swiftUITextContent`. Can't come up with a better name